packages/core/src/components/forms/inputGroup.tsx

```diff
@@ -156,6 +164,12 @@ export class InputGroup extends AbstractPureComponent<InputGroupProps, InputGrou
         }
     }
 
+    private handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const value = event.currentTarget.value;
+        this.props.onChange?.(event);
+        this.props.onValueChange?.(value/*, this.inputElement*/);
```

Comment:
`NumericInput` also passes back reference to the `inputElement`.
Wasn't sure if I should copy that pattern, not sure what it's for.
https://github.com/palantir/blueprint/blob/95b7fea60e08c86a7d406441f268bbd1a7a6a96e/packages/core/src/components/forms/numericInput.tsx#L642

Reply:
@adidahiya do you know why `inputElement` is an arg in `NumericInput`'s `onValueChange` callback?

Would you like me to do the same here?

Reply:
@thasner it was requested here: https://github.com/palantir/blueprint/issues/3943. I think it's a good idea to add it for your new prop as well.

Reply:
done in [86881c0](https://github.com/palantir/blueprint/pull/6290/commits/86881c0b1b9ca7722e10016572e6d77cb09620dd)